Exploration/README.md

```diff
@@ -22,3 +22,14 @@ approved by Launch Pad tech leads!
   is an overview of (deep) neural networks, one of the most popular techniques
   for machine learning. It's written by Michael Nielsen, also known for one
   of the standard quantum computing textbooks.
+
+## Coding Practice
+
+* [LeetCode](https://leetcode.com) is probably the most popular website for coding
+  challenges and comes highly recommended for interview preparation - big companies
+  often take interview questions directly from LeetCode!
+* [Project Euler](https://projecteuler.net/) is a very different style of coding
```

Comment:
we could also feature @iKevinY 's project here 😋 

https://github.com/iKevinY/EulerPy

Reply:
I'm in the process of fixing the Homebrew build right now! 😛

Reply:
Ew no pipenv 😢

Reply:
@rwblickhan In my defence, I wrote it 5 years ago 😅

Reply:
Not too late to change it 😛